reconcile/glitchtip_project_alerts/integration.py

```diff
@@ -103,8 +103,42 @@ def fetch_desired_state(
                             for recp in alert.recipients
                         ],
                     )
-                    for alert in glitchtip_project.alerts or []
-                ],
+                )
+            if glitchtip_project.jira and gjb_alert_url:
+                if not (jira_project_key := glitchtip_project.jira.project):
+                    if glitchtip_project.jira.board:
+                        jira_project_key = glitchtip_project.jira.board.name
+
+                # this shouldn't happen, because our schemas enforce it
+                if not jira_project_key:
+                    raise ValueError(
+                        "Either jira.project or jira.board must be set for Jira integration"
+                    )
+
+                url = f"{gjb_alert_url}/{jira_project_key}"
+                if gjb_token:
+                    url += f"?token={quote(gjb_token)}"
+
+                alerts.append(
+                    ProjectAlert(
+                        name="Jira-integration",
```

Comment:
Since this value is hard coded, will duplicate `name` causing issues?

Reply:
Good point. The integration will raise an error if someone uses the reserved alert name. See [eb3091e](https://github.com/app-sre/qontract-reconcile/pull/3880/commits/eb3091ed23b470e5ee50873a4089080e89be2efc)